src/index.ts

```diff
@@ -1,11 +1,21 @@
-import { createApp } from './app'
+import { createIdentityApp, createMatchingApp } from './app'
 import { DatabaseWrapper } from './databaseWrapper'
 
 const verifyServiceProviderHost = process.env.VERIFY_SERVICE_PROVIDER_HOST || 'http://localhost:50400'
 const databaseConnectionString = process.env.DATABASE_CONNECTION_STRING || 'postgesql://localhost:5432/stub_rp_test'
-const serviceEntityId = process.env.ENTITY_ID || null
+const serviceEntityId = process.env.ENTITY_ID
+const matching = process.env.MATCHING || false
 
-const server = createApp(verifyServiceProviderHost, DatabaseWrapper.getDatabaseWrapper(databaseConnectionString), serviceEntityId).listen(process.env.PORT || 3200, function () {
-  console.log(`Entity Id set to ${serviceEntityId !== null ? serviceEntityId : 'null (will use verify service provider default)'}`)
-  console.log(`Stub RP app listening on port ${server.address().port}!`)
-})
+if (matching) {
+  console.log(`Starting the app in a matching mode`)
```

Comment:
we probably should use 'debug' for this logging

Reply:
Then it's not going to show in the console. I think this is useful to see when starting the app (using `./startup` for instance)